ui/components/PromptComponent.js

```diff
@@ -64,14 +64,14 @@ const styles = (theme) => ({
   },
 });
 
-const PromptActionButton = styled(Button)(({ theme, promptVariant }) => ({
+const PromptActionButton = styled(Button)(({ theme }) => ({
   margin: theme.spacing(0.5),
   padding: theme.spacing(1),
   borderRadius: 5,
-  backgroundColor: theme.palette.secondary[promptVariant || 'primary'],
+  backgroundColor: theme.palette.secondary.focused,
   color: '#fff',
   '&:hover': {
-    backgroundColor: theme.palette.secondary[promptVariant || 'primary'],
+    backgroundColor: theme.palette.secondary.focused,
```

Comment:
@vaibhavpnimkar LGTM thanks, only one things as we now keeping background color same even on hover so we can remove this background color line on hover as this will also result same